evalml/tests/data_checks_tests/test_data_checks.py

```diff
@@ -383,51 +383,19 @@ def test_default_data_checks_regression(input_type, data_checks_input_dataframe)
     if input_type == "ww":
         y = ww.init_series(y)
         y_no_variance = ww.init_series(y_no_variance)
-    null_leakage = [
```

Comment:
The problem here is that the actual mutual information calculation changed this release so the scores are now 0.6 as opposed to >0.95 and there's no way to change that from the default data checks.

Reply:
should we put up an issue to track adding this coverage back? I'm pretty meh on this since we already have coverage on the unit tests for each check. It's also very painful to find or manipulate the existing test data to fail all these checks.